src/_guides/language/effective-dart/usage.md

```diff
@@ -688,27 +688,64 @@ constructor, you may need to do the "private field, public getter" pattern, but
 don't reach for that until you need to.
 
 
-### CONSIDER using `=>` for short members whose body is a single return statement.
+### CONSIDER using `=>` for simple members.
 
 In addition to using `=>` for function expressions, Dart also lets you define
-members with them. They are a good fit for simple members that just calculate
+members with it. That style is a good fit for simple members that just calculate
 and return a value.
 
 {:.good-style}
 <?code-excerpt "misc/lib/effective_dart/usage_good.dart (use-arrow)"?>
 {% prettify dart %}
-get width => right - left;
-bool ready(num time) => minTime == null || minTime <= time;
-containsValue(String value) => getValues().contains(value);
+double get area => (right - left) * (bottom - top);
+
+bool isReady(num time) => minTime == null || minTime <= time;
+
+String capitalize(String name) =>
+    "${name[0].toUpperCase()}${name.substring(1)}";
+{% endprettify %}
+
+People *writing* code seem to love `=>`, but it's very easy to abuse it and end
+up with code that's hard to *read*. If your declaration is more than a couple of
+lines or contains deeply nested expressions&mdash;cascades and conditional
+operators are common offenders&mdash;do yourself and everyone who has to read
+your code a favor and use a block body and some statements.
+
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/usage_good.dart (arrow-long)"?>
+{% prettify dart %}
+Treasure openChest(Chest chest, Point where) {
+  if (_opened.containsKey(chest)) return null;
+
+  var treasure = new Treasure(where);
+  treasure.addAll(chest.contents);
+  _opened[chest] = treasure;
+  return treasure;
+}
 {% endprettify %}
 
-Members that don't fit on one line can still use `=>`, but if you find yourself
-cramming a single expression into several continued lines, it is probably
-cleaner to just use a curly body with an explicit `return`.
+{:.bad-style}
+<?code-excerpt "misc/lib/effective_dart/usage_bad.dart (arrow-long)"?>
+{% prettify dart %}
+Treasure openChest(Chest chest, Point where) =>
+    _opened.containsKey(chest) ? null : _opened[chest] = new Treasure(where)
+      ..addAll(chest.contents);
+{% endprettify %}
+
+You can also use `=>` on void members that don't return a value. This is
+idiomatic for small setters that have a corresponding getter also using `=>` and
```

Comment:
I'm not sure what the "and where..." part means. It could be a consequence of what comes before, or an alternative. Can we just simplify it? Maybe:

This is idiomatic when a setter is small and has a corresponding getter that uses `=>`.

Reply:
Fixed